packages/reload-middleware/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "@sap-ux/reload-middleware",
+    "description": "Reload middleware",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/reload-middleware"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Apreview-middleware"
+    },
+    "version": "0.0.1",
+    "author": "@SAP/ux-tools-team",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "start:fixture": "ui5 serve --config test/fixtures/simple-app/ui5.yaml",
+        "build": "npm-run-all -l -p build:middleware",
+        "build:middleware": "tsc --build",
+        "watch": "tsc --watch",
+        "clean": "rimraf --glob dist test/test-output coverage *.tsbuildinfo playwright-report",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "ui5.yaml",
+        "!dist/*.map",
+        "!dist/**/*.map"
+    ],
+    "dependencies": {
+        "ui5-middleware-websocket": "3.0.1",
+        "@sap-ux/logger": "workspace:*",
+        "@types/ws": "8.5.10"
```

Comment:
Move this to `devDepenencies` as types are not needed at runtime.

Reply:
moved